cypher-manual/functions.md

```diff
@@ -31,7 +31,7 @@ This section contains the list of supported functions.
  | `toBoolean`  | `toBoolean(value: any) -> (boolean)`                                                 | Converts the argument to a boolean.                                                                                                                                                                                                      |
  | `toFloat`    | `toFloat(value: any) -> (float)`                                                     | Converts the argument to a floating point number.                                                                                                                                                                                        |
  | `toInteger`  | `toInteger(value: any) -> (integer)`                                                 | Converts the argument to an integer.                                                                                                                                                                                                     |
- | `toString`   | `toString(value: any) -> (string)`                                                   | Converts the argument to a string.                                                                                                                                                                                                       |
```

Comment:
Could you please align the `|` symbols for this table? For most of table even the line end are perfectly aligned, except this and the next table. As you only touched this, it is totally okay if you only beautify this one.

I think this should do the work, but please double check:
```
 | Name         | Signature                                                                                                | Description                                                                                                                                                                                                                                      |
 | ------------ | -------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | `startNode`  | `startNode(relationship: Relationship) -> (Node)`                                                        | Returns the starting node of a relationship.                                                                                                                                                                                                     |
 | `endNode`    | `endNode(relationship: Relationship) -> (Node)`                                                          | Returns the destination node of a relationship.                                                                                                                                                                                                  |
 | `degree`     | `degree(node: Node) -> (integer)`                                                                        | Returns the number of relationships (both incoming and outgoing) of a node.                                                                                                                                                                      |
 | `head`       | `head(list: List[any]) -> (any)`                                                                         | Returns the first element of a list.                                                                                                                                                                                                             |
 | `last`       | `last(list: List[any]) -> (any)`                                                                         | Returns the last element of a list.                                                                                                                                                                                                              |
 | `properties` | <code>properties(value: Node\|Relationship) -> (Map[string, any])</code>                                 | Returns the property map of a node or a relationship.                                                                                                                                                                                            |
 | `size`       | `size(value: any) -> (integer)`                                                                          | Returns the number of elements in the value. When given a **list** it returns the size of the list. When given a string it returns the number of characters. When given a path it returns the number of expansions (relationships) in that path. |
 | `toBoolean`  | `toBoolean(value: any) -> (boolean)`                                                                     | Converts the argument to a boolean.                                                                                                                                                                                                              |
 | `toFloat`    | `toFloat(value: any) -> (float)`                                                                         | Converts the argument to a floating point number.                                                                                                                                                                                                |
 | `toInteger`  | `toInteger(value: any) -> (integer)`                                                                     | Converts the argument to an integer.                                                                                                                                                                                                             |
 | `toString`   | `toString(value: any) -> (string)`                                                                       | Converts the argument to a string.                                                                                                                                                                                                               |
 | `toString`   | <code>toString(value: String\|Number\|Date\|LocalTime\|LocalDateTime\|Duration\|Bool) -> (string)</code> | Converts the argument to a string.                                                                                                                                                                                                               |
 | `type`       | `type(relationship: Relationship) -> (string)`                                                           | Returns the type of a relationships as a character string.                                                                                                                                                                                       |
 | `assert`     | `assert(value: any) -> ()`                                                                               | Raises an exception if the given argument is not `true`.                                                                                                                                                                                         |
 | `id`         | <code>id(value: Node\|Relationship) -> (integer)</code>                                                  | Returns identifier for a given node or relationship. The identifier is generated during the initialization of a node or a relationship  and will be persisted through the durability mechanism.                                                  |
 | `timestamp`  | `timestamp() -> (integer)`                                                                               | Returns the difference, measured in microseconds, between the current time and midnight, January 1, 1970 UTC.                                                                                                                                    |
 | `counter`    | `counter(name: string, initial-value: integer, increment: integer = 1) -> (integer)`                     | Generates integers that are guaranteed to be unique within a single query for a given counter name.  The increment parameter can be any integer besides zero.                                                                                    |
```